_pages/en/it-picture-medium-term.md

```diff
@@ -113,7 +122,10 @@ The build statuses of projects are visible to the IT branch.
 Teams have access to adequate test data to run all the automated tests for a project.
 
 Enterprise architecture (EA) is designed to enable teams to test, deploy, and change systems without dependencies on other teams requiring less communication to get work done.
-EA is an agility enabler by empowering teams to choose tools and technologies, rapidly responding to new standards adoption, as well as providing light reference architectures as guardrails to product teams to assure compliance.
+EA is an agility enabler by empowering teams to choose tools and technologies, rapidly responding to new standards adoption, as well as providing light reference architectures, vetted by IT Security, as guardrails to product teams to assure compliance.
+
+EA enables ESDC to re-use investments by guiding product teams towards the desired end state of ESDC's digital platform vision (TODO: link to digital platform vision).
+EA standards have moved towards established core services of the digital platform that product teams are required to use for compliance.
```

Comment:
This repeats what's above, i think

Reply:
I will reword. But then I think to remove the following line as I don't think it's an EA function
> Enterprise architecture (EA) is designed to enable teams to test, deploy, and change systems without dependencies on other teams requiring less communication to get work done.

The main idea is that EA does not only do light reference architectures, but also guides investment decisions by re-using core services (actual implementations). These core services are what are the standards. Not the underlying technology behind it (e.g. if you want to post an add on LinkedIn, you use one of their core service, you don't build a new one)